packages/components/src/test/tree/controlled/PresentationTreeRenderer.test.tsx

```diff
@@ -151,6 +153,36 @@ describe("PresentationTreeRenderer", () => {
     expect(dialog).to.be.null;
   });
 
+  it("does not render filter dialog when tree model does not find a matching node", async () => {
+    const { visibleNodes, nodeLoader } = setupTreeModel((model) => {
+      model.setChildren(
+        undefined,
+        [createTreeModelNodeInput({ id: "A", item: { filtering: { descriptor: createTestContentDescriptor({ fields: [] }), ancestorFilters: [] } } })],
+        0,
+      );
+    });
+
+    // stub getNode method to make it return undefined when onFilterClick() is called.
+    nodeLoader.modelSource.getModel = () => {
+      return { getNode: sinon.stub().returns(undefined) } as unknown as TreeModel;
+    };
```

Comment:
I think it should be:
```suggestion
    sinon.stub(nodeLoader.modelSource, "getModel").returns({
      getNode: () => undefined 
    } as unknown as TreeModel};
```

or 

```suggestion
    sinon.stub(nodeLoader.modelSource.getModel(), "getNode").returns(undefined);
```

Reply:
Will stick to latter one, seems more neat